Neos.ContentRepository.Core/Classes/DimensionSpace/DimensionSpacePoint.php

```diff
@@ -90,7 +90,13 @@ final public static function fromLegacyDimensionArray(array $legacyDimensionValu
 
     final public static function fromUriRepresentation(string $encoded): self
     {
-        return self::instance(json_decode(base64_decode($encoded), true));
+        parse_str(base64_decode(str_replace( '~', '=', $encoded)), $parsed);
+        return self::instance($parsed);
+    }
+
+    final public function toUriRepresentation(): string
+    {
+        return str_replace('=', '~', base64_encode(http_build_query($this->coordinates)));
```

Comment:
Yikes, I might need ot understand how this is supposed to be used? Why not just jsonify?